content/reference-transaction-format.md

```diff
@@ -541,14 +542,14 @@ An offer in the ledger can be fulfilled either by additional OfferCreate transac
 
 You can create an offer so long as you have at least some (any positive, nonzero amount) of the currency specified by the `TakerGets` parameter of the offer. The offer sells as much of the currency as you have, up to the `TakerGets` amount, until the `TakerPays` amount is satisfied. An offer cannot place anyone in debt.
 
-It is possible for an offer to become temporarily or permanently *unfunded*:
+It is possible for an offer to become temporarily or permanently _unfunded_:
 
 * If the creator no longer has any of the `TakerGets` currency.
-  * The offer becomes funded again when the creator obtains more of that currency.
+    * The offer becomes funded again when the creator obtains more of that currency.
```

Comment:
Yes. 4 spaces are required to trigger an indent; there were 2. (I think this might also be a difference between some Markdown parsers.)